Lukem/index page learn (#212)
moondance-labs/tanssi-docs: learn/tanssi/network-features/index.md

---
title: Network Features

> description: Learn about Tanssi general network features such as Staking, Governance and Treasury, and how each help Tanssi becoming a fully decentralized Appchain orchestrator for the Web3 ecosystem.

Reviewer: ```suggestion
description: Learn about Tanssi general network features such as staking, governance, and treasury, and how each help Tanssi becoming a fully decentralized network orchestrator for the Web3 ecosystem.
```

> description: Learn about Tanssi general network features such as staking, governance, and treasury, and how each help Tanssi becoming a fully decentralized network orchestrator for the Web3 ecosystem.

Author: I think since the docs have been created, haha.

icon: octicons-cpu-24
template: index-page.html
---

# Network Features

> The network features in Tanssi are designed to provide the foundational mechanisms that support decentralization, security, and scalability. From securing appchains through staking to future capabilities, this section outlines the core elements that empower Tanssi’s appchain infrastructure.

Reviewer: ```suggestion
The network features in Tanssi are designed to provide the foundational mechanisms that support decentralization, security, and scalability. From securing networks through staking to future capabilities, this section outlines the core elements that empower Tanssi’s network infrastructure.
```

---
title: Network Features
description: Learn about Tanssi general network features such as Staking, Governance and Treasury, and how each help Tanssi becoming a fully decentralized Appchain orchestrator for the Web3 ecosystem.
icon: material-blur
template: index-page.html
---

# Network Features

The network features in Tanssi are designed to provide the foundational mechanisms that support decentralization, security, and scalability. From securing appchains through staking to future capabilities, this section outlines the core elements that empower Tanssi’s appchain infrastructure.

> 

Reviewer: This is a super teeny tiny nit pick. But it's good to be aware of the spacing in the markdown files and only have one line break between elements.

```suggestion
```
